### ISSUE TITLE
Builder automation:  option for manual triggering

### ISSUE THREAD

**Issue by bkmartinjr** [chanzuckerberg/cellxgene-census#626]
It would be nice to have the ability to manually trigger the production (Batch) build automation. Use cases:
* test builds
* ad hoc builds triggered by an external need (e.g. upstream data changes)

**Comment by metakuni** [comment]
Replace with: Do test run in dev once 3.1 is in place (as discussed with @atolopko-czi & @ebezzi )

**Comment by atolopko-czi** [comment]
> Replace with: Do test run in dev once 3.1 is in place (as discussed with @atolopko-czi & @ebezzi )

Note that the issue is still valid and worth doing, but it is not required work for the "3.1 datasets" epic, since we can first test the build in dev and then just let the prod run trigger on the normal schedule. And if we want to trigger earlier, this can be simply done from the AWS StepFn console.

**Comment by bkmartinjr** [comment]
Removing the `3.1-dataset-schema` label from this issue, as it is unrelated.
